Source/React.cpp

```diff
@@ -170,6 +182,20 @@ PeleC::react_state(
         auto const& mask = dummyMask.array(mfi);
         auto const& fc = fctCount.array(mfi);
 
+        if (use_chem_mask) {
+          amrex::ParallelFor(
+            bx, [=] AMREX_GPU_DEVICE(int i, int j, int k) noexcept {
+              amrex::XDim3 point;
+              point.x = plo[0] + (i + 0.5) * dx[0];
+              point.y = plo[1] + (j + 0.5) * dx[1];
+              point.z = plo[2] + (k + 0.5) * dx[2];
+
+              if (Chem_Masked_Region.contains(point)) {
+                mask(i, j, k) = -1;
```

Comment:
You can then make this a ternary expression: `mask(i, j, k) = Chem_Masked_Region.contains(iv)? -1 : 0;`. It's also weird to me that `mask` is -1 when masked. Should it be 0? And then 1 for when unmasked. That would fit my mental definition of true/false and other uses of the term "mask" in amrex codes.